assets/scripts/script.js

```diff
@@ -63,6 +75,33 @@ $(document).ready(function () {
 
     }
 
+    function airqualityMetrics(cityName) {
+
+        // var queryURL = "https://api.airvisual.com/v2/city?city=" + cityName+"&state=new york&country=usa&key=bc4dec27-7130-4a22-88ca-f37ecbcfc5f9";
+        var queryURL = "https://api.weatherbit.io/v2.0/current/airquality?city=" + cityName +"&country=US&key=e5f946832cc34874b9250ae7016416e0";
+
+        $.ajax({
+            type: "GET",
+            url: queryURL,
+            dataType: "json",
+            success: function (response) {
+                console.log(response);
+
+                var CO = $(".airInfo1").text("Carbon Monoxide (CO) : " + response.data[0].co);
+                var no2 = $(".airInfo2").text("Nitrogen Dioxide (NO2) : " + response.data[0].no2);
+                var o3 = $(".airInfo3").text("Ozone (O3) : " + response.data[0].o3);
+                var pollentype = $(".airInfo4").text("Pollen Type: " + response.data[0].predominant_pollen_type);
+                var moldLevel = $(".airInfo5").text(response.data[0].predominant_pollen_type+" Level: " + response.data[0].mold_level);
+                var pollenGrass = $(".airInfo6").text("Pollen Level Grass: " + response.data[0].pollen_level_grass);
+                var pollentree = $(".airInfo7").text("Pollen Level Tree: " + response.data[0].pollen_level_tree);
+                var pollenweed = $(".airInfo7").text("Pollen Level Weed: " + response.data[0].pollen_level_weed);
+
+                $("#air-metric").append(CO, no2, o3, pollentype, pollenGrass, pollentree, pollenweed, moldLevel);
```

Comment:
The data points are appending to the end of the Air Quality Metrics div, instead of building inside of it. Since you are adding all of the <p> tags into the html, you shouldn't need to append anything. You also have .airInfo7 duplicated on line 97.